.circleci/config.yml

```diff
@@ -20,3 +20,58 @@ jobs:
             pip3 install -r requirements.txt
             pycodestyle
             pytest -k unit # only run unit tests at this stage
+
+  deploy:
+    working_directory: ~/sparql-loader
+    docker:
+      - image: circleci/python:3.6.4  # primary container
+    steps:
+      - checkout # checkout source code to the working directory
+      - run: sudo chown -R circleci:circleci /usr/local/bin
+      - run: sudo chown -R circleci:circleci /usr/local/lib/python3.6/site-packages
```

Comment:
I found that there was code in `dev_server.py` that required 3.7. Maybe it's only required for the dev server, though. Are y'all aware of any reasons why 3.6 won't be sufficient to deploy the sparql loader, @aaron-collier @justinlittman?

Reply:
WFM